src/resolvers/Mutation/createEvent.ts

```diff
@@ -174,7 +170,71 @@ export const createEvent: MutationResolvers["createEvent"] = async (
     }
   }
      */
+  if (!createdEvent) {
```

Comment:
This is not sufficient. createdEvent can be empty array right if startDate > endDate. Can you please tell is this condition handled in some directives and if not then how can you throw error if you have allowed client to provide this data

Reply:
we are already checking this,  if startDate > endDate we are returning user with the error that start date cannot be greater than end date